frontend/src/scenes/surveys/SurveyView.tsx

```diff
@@ -103,6 +106,14 @@ export function SurveyView({ id }: { id: string }): JSX.Element {
                             </>
                         }
                     />
+                    {showSurveyAppWarning && (
+                        <LemonBanner type="error">
+                            Surveys requires the{' '}
+                            <a onClick={() => surveyPlugin?.id && editPlugin(surveyPlugin.id)}>survey app</a> to be
+                            enabled. Enable and save it, and make sure you have the "opt_in_site_apps" setting in your
```

Comment:
Does this sound too direct and bossy? @joethreepwood

Reply:
```suggestion
                            enabled. You also need to make sure you have the "opt_in_site_apps" setting in your
```
Not at all. This sounds a little simpler, however.